Classes/Domain/Model/Facet.php

```diff
@@ -21,6 +21,13 @@ class Facet implements \JsonSerializable
      */
     protected $value = '';
 
+    /**
+     * Type of the input field.
+     *
+     * @var string
+     */
+    protected $inputType = '';
```

Comment:
does a default make sense here to guarantee, that we have a valid value?

Reply:
According to our openAPI specs inputType is optional. This is mainly because I had to keep compatibility with the old backend. Therefore the frontend component (or any other client) needs to be able to handle no value anyway.

Reply:
Ok, maybe just ensure it’s set in the backend always. Makes error handling easier